apps/web/src/services/cost-explorer/lib/helper.ts

```diff
@@ -13,14 +13,22 @@ export const getTimeUnitByGranularity = (granularity: Granularity): TimeUnit =>
     if (granularity === GRANULARITY.MONTHLY) return 'month';
     return 'day';
 };
+export const getPeriodByGranularity = (granularity: Granularity, period: Period): Period => {
+    if (granularity === GRANULARITY.DAILY) {
+        return {
+            start: dayjs.utc(period.start).startOf('month').format('YYYY-MM-DD'),
+            end: dayjs.utc(period.end).endOf('month').format('YYYY-MM-DD'),
+        };
+    }
+    return period;
+};
```

Comment:
question: In "this Month" case, Can I use 'endof' as well? It might need to be specified until to today or something like that.